pkg/informer/informer.go

```diff
@@ -110,7 +110,7 @@ var _ genericInformerBase[cache.SharedIndexInformer, cache.GenericLister] = (inf
 // logical-cluster-unaware (== single cluster, standard kube) informers.
 type GenericDiscoveringDynamicSharedInformerFactory[Informer cache.SharedIndexInformer, Lister genericListerBase, GenericInformer genericInformerBase[Informer, Lister]] struct {
 	newInformer func(gvr schema.GroupVersionResource, resyncPeriod time.Duration, indexers cache.Indexers) GenericInformer
-	filterFunc  func(interface{}) bool
+	filterFunc  func(schema.GroupVersionResource, interface{}) bool
```

Comment:
unrelated to the PR ?

Reply:
It is related to this PR. Used when exposing the `logicalclusters` in the Syncer VW, in order to exclude it from the registered ddsif resources handlers.
cf https://github.com/kcp-dev/kcp/pull/2675/files#diff-b043590efbb8609f4eced1377afbe4f51358c5c628fb79c0fdd24e8f9d553551R261

Reply:
Removed since the logical clusters are not required anymore